tools/releaseBuild/LinuxBuild.yml

```diff
@@ -0,0 +1,122 @@
+resources:
+- repo: self
+  clean: true
+jobs:
+
+- job: Job_1
```

Comment:
Give the job a meaningful name, like `Build`

Reply:
updated.